Formula/portable-openssl.rb

```diff
@@ -39,6 +39,7 @@ def arch_args
       {
         :x86_64 => %w[linux-x86_64],
         :i386  => %w[linux-generic32],
+        :arm => %w[linux-armv4],
```

Comment:
This is the only flag OpenSSL offers for ARM. 

As the minimum CPU, maybe ARMv7 as that is what the oldest docker image of Raspberry PI I obtained uses.